SniffMeet/SniffMeet/Source/Auth/CreateProfile/View/ProfileSetupView.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  ProfileSetupView.swift
+//  SniffMeet
+//
+//  Created by 윤지성 on 11/9/24.
+//
+
+import UIKit
+import PhotosUI
```

Comment:
P4. 둘 다 빌트 인 프레임워크니 알파벳 순서를 지키면 좋을거 같습니다.
```suggestion
import PhotosUI
import UIKit
```

Reply:
알파벳 순서 지켰습니다! 다시 한번 확인해주세요!